### PR TITLE
added method `update_note` to alerts

### PR DESCRIPTION
**Pull request by kiran-chaudhary**
### Description of Change ###
Added method `update_note`

### Issues Resolved ###

- closes #INTEG-1427

### Testing Procedure ###
`sdk.alerts.update_note("alert-id", "note")`

### PR Checklist ###
Did you remember to do the below?

- [x] Add unit tests to verify this change
- [x] Add an entry to CHANGELOG.md describing this change
- [x] Add docstrings for any new public parameters / methods / classes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Added
 
+- `sdk.alerts.update_note` to add or update note regarding an alert.
```
Comment (unparalleled-js) [inline CHANGELOG.md:15]: Can you end the method name with `()`?
`sdk.alerts.update_note()`

File: src/py42/clients/alerts.py [hunk 2]
```diff
@@ -80,3 +80,12 @@ def update_state(self, status, alert_ids, note=None):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.update_state(status, alert_ids, note=note)
+
+    def update_note(self, alert_id, note):
+        """Add/update  a note to an alert.
```
Comment (unparalleled-js) [inline src/py42/clients/alerts.py:85]: There is an extra space between `Add/update` and `a note...`

File: src/py42/clients/alerts.py [hunk 3]
```diff
@@ -80,3 +80,12 @@ def update_state(self, status, alert_ids, note=None):
...
+
+        Args:
+            alert_id (str): The identification number of an alert to add a note to.
+            note (str): User note regarding the alert, in less than 2000 characters.
+        """
```
Comment (unparalleled-js) [inline src/py42/clients/alerts.py:90]: Missing `returns:` block

Comment (kiran-chaudhary) [reply src/py42/clients/alerts.py:93]: done.

File: src/py42/clients/alerts.py (hunk 2 above)
Comment (kiran-chaudhary) [reply src/py42/clients/alerts.py]: done.

File: CHANGELOG.md (hunk 1 above)
Comment (kiran-chaudhary) [reply CHANGELOG.md]: Done.

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**